src/components/ArticleData.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import AvatarImage from "./AvatarImage";
+import '../css/ArticleData.css'
+
+export default class ArticleData extends React.Component {
+    render() {
+        return (
+            <div className="ArticleData">
+                <div className={"flexbox"}>
```

Comment:
On top of this line you use this syntax `className="ArticleData"` and here `className={"flexbox"}`. Please keep consistent.

Reply:
Thanks, I'll fix it.